frida_tools/tracer.py

```diff
@@ -53,6 +52,7 @@ def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
             parser.add_option("-s", "--include-debug-symbol", help="include DEBUG_SYMBOL", metavar="DEBUG_SYMBOL",
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_debug_symbol,))
             parser.add_option("-q", "--quiet", help="do not format agent's output", action='store_true')
+            parser.add_option("-o", "--output", help="dump agent messages to file", metavar="OUTPUT", type='string')
```

Comment:
Perhaps just: `dump messages to file`, as it's probably unnecessary to mention `agent` here. (If so we should drop it for `--quiet` also.)